Assets/Mirror/Weaver/Extensions.cs

```diff
@@ -1,4 +1,4 @@
-using Mono.Cecil;
+using Mono.MirrorCecil;
```

Comment:
Did you recompile cecil yourself or what?

Reply:
Yeah pattern replace all cs files, and then pattern replace in the csproj files for rootnamespace and final assembly name. Could be automated if we ever needed that